### PR TITLE
[#11418] set lidar_timestamp in velodyne32_parser

### LINKED ISSUE #11418: velodyne driver: lidar_timestamp is not set for VLP32C and HDL32E in velodyne32_parser.cc
Here is the code snippet where measurement_time is set:

```
void Velodyne32Parser::GeneratePointcloud(
    const std::shared_ptr<VelodyneScan>& scan_msg,
    std::shared_ptr<PointCloud> out_msg) {
  // allocate a point cloud with same time and frame ID as raw data
  out_msg->mutable_header()->set_frame_id(scan_msg->header().frame_id());
  out_msg->set_height(1);
  out_msg->mutable_header()->set_sequence_num(
      scan_msg->header().sequence_num());
  gps_base_usec_ = scan_msg->basetime();

  size_t packets_size = scan_msg->firing_pkts_size();
  if (config_.model() == VLP32C) {
    for (size_t i = 0; i < packets_size; ++i) {
      UnpackVLP32C(scan_msg->firing_pkts(static_cast<int>(i)), out_msg);
      last_time_stamp_ = out_msg->measurement_time();
    }
  } else {
    for (size_t i = 0; i < packets_size; ++i) {
      Unpack(scan_msg->firing_pkts(static_cast<int>(i)), out_msg);
      last_time_stamp_ = out_msg->measurement_time();
      ADEBUG << "stamp: " << std::fixed << last_time_stamp_;
    }
  }
  if (out_msg->point().empty()) {
    // we discard this pointcloud if empty
    AERROR << "All points is NAN!Please check velodyne:" << config_.model();
  }
  // set default width
  out_msg->set_width(out_msg->point_size());
}

void Velodyne32Parser::UnpackVLP32C(const VelodynePacket& pkt,
                                    std::shared_ptr<PointCloud> pc) {
...

      // compute time
      uint64_t timestamp = static_cast<uint64_t>(GetTimestamp(
          basetime, (*inner_time_)[i][laser_id], static_cast<uint16_t>(i)));

      if (laser_id == SCANS_PER_BLOCK - 1) {
        // set header stamp before organize the point cloud
        pc->set_measurement_time(static_cast<double>(timestamp) / 1e9);
      }

...
}

void Velodyne32Parser::Unpack(const VelodynePacket& pkt,
                              std::shared_ptr<PointCloud> pc) {
...

      if (laser_id == SCANS_PER_BLOCK - 1) {
        // set header stamp before organize the point cloud
        pc->set_measurement_time(static_cast<double>(timestamp) / 1e9);
      }

...
}
```

But out_msg->header()->lidar_timestamp() never gets set.

### PR DESCRIPTION
**Pull request by jilinzhou**
- The lidar_timestamp of outgoing point cloud msg was not set.
- Also fixes consistent warning msgs in GetTimestamp() for vlp32c packets.
- Verified with our in-house vlp 32c
- Fixes #11418

### CODE REVIEW

File: modules/drivers/velodyne/parser/velodyne32_parser.cc [hunk 1]
```diff
@@ -98,14 +136,10 @@ void Velodyne32Parser::UnpackVLP32C(const VelodynePacket& pkt,
...
-          basetime, (*inner_time_)[i][laser_id], static_cast<uint16_t>(i)));
-
-      if (laser_id == SCANS_PER_BLOCK - 1) {
-        // set header stamp before organize the point cloud
-        pc->set_measurement_time(static_cast<double>(timestamp) / 1e9);
-      }
+      // compute the actual timestamp associated with the data point at data
+      // block i and laser_id
+      uint64_t timestamp = GetTimestamp(basetime, (*inner_time_)[i][laser_id],
+                                        static_cast<uint16_t>(i));
```
Comment (jinghaomiao) [inline modules/drivers/velodyne/parser/velodyne32_parser.cc:142]: Please reformat the code here.

**Comment by jinghaomiao** [comment]
@jilinzhou thanks for the pull request. It's related to driver and I would suggest that you provide the supporting results after the fix. Thank you!

File: modules/drivers/velodyne/parser/velodyne32_parser.cc (hunk 1 above)
Comment (jilinzhou) [reply modules/drivers/velodyne/parser/velodyne32_parser.cc:142]: I formatted the code with apollo/.clang-format. Anything specific here is not following the coding format requirements?

**Comment by jilinzhou** [comment]
> @jilinzhou thanks for the pull request. It's related to driver and I would suggest that you provide the supporting results after the fix. Thank you!

Screenshot from 2020-07-08 08-47-31

**Pull request merged**